charters/QueryService/charter.md

```diff
@@ -0,0 +1,71 @@
+
+# Query Service
+
+
+## Description
+The Query Service is an Application Programming Interface (API) for querying Human Cell Atlas (HCA) bundle metadata with different query languages.
+
+## Objectives
+1. Build, operate, and maintain a metadata API query service that allows a variety of queries to be performed against bundle metadata stored in HCA Data Coordination Platform (DCP) Data Storage Service (DSS)
+1. Develop a query language specification and metadata Extract, Transform, Load (ETL) pattern to accommodate intuitive querying
+1. Conduct user experience (UX) research to validate query language usability, understand remaining usability gaps in DCP data searchability and discoverability
+1. Develop a scalable public API for external indexing and other DSS event consumers
+1. Migrate existing ElasticSearch (ES) query subsystem to this public API
+1. Deploy all query systems as stand-alone, modular services
+1. Make DCP index and query systems agnostic to DCP schema evolution
+
+## In-scope
+* Common index event bus and deployment infrastructure
+    * Finalize support for multiple metadata indexers in DSS
+    * Expose event bus for bundle lifecycle event subscriptions
+    * Scalable, fault-tolerant queuing mechanism: Amazon Web Services (AWS) Simple Queuing Service (SQS) push or similar
```

Comment:
This first statement seems heavily implementation specific; why is this being called out? I couldn't figure out why I need this information

Reply:
✅